src/common/loader_utils.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+  'use strict';
+
+  RAML.LoaderUtils = {
+
+    allowedRamlOrigin : function(options) {
```

Comment:
Code seems inconsistently styled all over this function scope.
- Remove space before `:` in function declaration.
- There should be a space before and after equal sign in variable declaration / assignment.
- Stick to 2-space indentation everywhere.